src/aind_data_transfer/jobs/basic_job.py

```diff
@@ -371,14 +412,19 @@ def _add_processing_to_metadata(
     def _encrypt_behavior_dir(self, temp_dir: Path) -> None:
```

Comment:
I don't grok this entirely, but you probably want to be encrypting the behavior videos dir, not the behavior dir.

Reply:
This is the legacy function. It looks for a "behavior" modality in the job args and writes to the temp_dir. I can clean this up during the refactor. This pr is mostly to update aind-data-schema so people can start using the updated modalities like ISI.